src/System.Management.Automation/engine/parser/Parser.cs

```diff
@@ -4514,7 +4514,7 @@ private MemberAst ClassMemberRule(string className, out List<Ast> astsOnError)
                 }
```

Comment:
@vexx32, your last commit had 2 failures in `PowerShell-CI-macos`
Testing Method Names can be Keywords.Permits class methods to be named after keywords
```powershell
You cannot call a method on a null-valued expression.
at <ScriptBlock>, /Users/vsts/agent/2.153.2/work/1/s/test/powershell/Language/Classes/Scripting.Classes.BasicParsing.Tests.ps1: line 472
472:         $Results = Invoke-PowerShell -Script $TestScript

```
Testing Method Names can be Keywords.Permits class methods to be named after DynamicKeywords
```powershell
You cannot call a method on a null-valued expression.
at <ScriptBlock>, /Users/vsts/agent/2.153.2/work/1/s/test/powershell/Language/Classes/Scripting.Classes.BasicParsing.Tests.ps1: line 520
520:         Invoke-PowerShell -Script $DefineKeyword

```